R/makeids.r

```diff
@@ -6,72 +6,41 @@
-									"ER30313","ER30343","ER30373","ER30399",
-									"ER30429","ER30463","ER30498","ER30535",
-									"ER30570","ER30606","ER30642","ER30689",
-									"ER30733","ER30806","ER33101","ER33201",
-									"ER33301","ER33401","ER33501","ER33601",
-									"ER33701","ER33801","ER33901","ER34001","ER34101")
+	id.list$ind.interview <- c("ER30001","ER30020","ER30043","ER30067","ER30091","ER30117",
+				   "ER30138","ER30160","ER30188","ER30217","ER30246","ER30283",
+				   "ER30313","ER30343","ER30373","ER30399","ER30429","ER30463",
+				   "ER30498","ER30535","ER30570","ER30606","ER30642","ER30689",
+				   "ER30733","ER30806","ER33101","ER33201","ER33301","ER33401",
+				   "ER33501","ER33601","ER33701","ER33801","ER33901","ER34001",
+				   "ER34101","ER34201")
 
-	id.list$ind.seq <- c(NA,"ER30021","ER30044","ER30068","ER30092","ER30118","ER30139",
-						 "ER30161","ER30189","ER30218","ER30247","ER30284","ER30314", 
-						 "ER30344","ER30374","ER30400","ER30430","ER30464","ER30499", 
-						 "ER30536","ER30571","ER30607","ER30643","ER30690","ER30734", 
-						 "ER30807","ER33102","ER33202","ER33302","ER33402","ER33502", 
-						 "ER33602","ER33702","ER33802","ER33902","ER34002","ER34102")
+	id.list$ind.seq <- c(    NA   ,"ER30021","ER30044","ER30068","ER30092","ER30118",
+			     "ER30139","ER30161","ER30189","ER30218","ER30247","ER30284",
+			     "ER30314","ER30344","ER30374","ER30400","ER30430","ER30464",
+			     "ER30499","ER30536","ER30571","ER30607","ER30643","ER30690",
+			     "ER30734","ER30807","ER33102","ER33202","ER33302","ER33402",
+			     "ER33502","ER33602","ER33702","ER33802","ER33902","ER34002",
+			     "ER34102","ER34202")
 	
 	# name of variable "relationship to head"
-	id.list$ind.head <- c("ER30003",
-						  "ER30022",
-						  "ER30045",
-						  "ER30069",
-						  "ER30093",
-						  "ER30119",
-						  "ER30140",
-						  "ER30162",
-						  "ER30190",
-						  "ER30219",
-						  "ER30248",
-						  "ER30285",
-						  "ER30315",
-						  "ER30345",
-						  "ER30375",
-						  "ER30401",
-						  "ER30431",
-						  "ER30465",
-						  "ER30500",
-						  "ER30537",
-						  "ER30572",
-						  "ER30608",
-						  "ER30644",
-						  "ER30691",
-						  "ER30735",
-						  "ER30808",
-						  "ER33103",
-						  "ER33203",
-						  "ER33303",
-						  "ER33403",
-						  "ER33503",
-						  "ER33603",
-						  "ER33703",
-						  "ER33803",
-						  "ER33903",
-						  "ER34003",
-						  "ER34103")
+	id.list$ind.head <- c("ER30003","ER30022","ER30045","ER30069","ER30093","ER30119",
+			      "ER30140","ER30162","ER30190","ER30219","ER30248","ER30285",
+			      "ER30315","ER30345","ER30375","ER30401","ER30431","ER30465",
+			      "ER30500","ER30537","ER30572","ER30608","ER30644","ER30691",
+			      "ER30735","ER30808","ER33103","ER33203","ER33303","ER33403",
+			      "ER33503","ER33603","ER33703","ER33803","ER33903","ER34003",
+			      "ER34103","ER34203")
```

Comment:
"ER34203" is not the right value for `relationship to head` in the 2013 wave. pretty hard bug to find. fixed.